### ISSUE TITLE
Including Bourbon and Neat as packages

### ISSUE THREAD

**Issue by tysongach** [18F/web-design-standards#1013]
Right now, we include the [Bourbon](http://bourbon.io) and [Neat](http://neat.bourbon.io) libraries as static files right within this repo, at [`/src/stylesheets/lib/`](https://github.com/18F/web-design-standards/tree/18f-pages-staging/src/stylesheets/lib).

Both of these libraries are distributed to multiple package managers, for example [RubyGems](https://rubygems.org/gems/bourbon) and [npm](https://www.npmjs.com/package/bourbon).

With libraries, the source files shouldn’t be changed and it’s generally best practice to use package managers for them; it sure makes updates easy. Is there a reason we aren’t using npm to pull in Bourbon and Neat? I mention npm, specifically, as it looks like we’re moving towards using Gulp for our build processes.

**Comment by msecret** [comment]
Hey @tysongach, I've actually been working on this here: https://github.com/18F/web-design-standards/pull/999/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2. 

One problem I ran into: I'm changing the include paths in the uswds gulp scss task to include the bourbon and neat paths, but when I'm using the src sass files on another project that uses the standards, the paths for bourbon and neat aren't there. 

Have you heard of a good way to solve that besides just using the full node_modules path (`@import "./node_modules/bourbon/..."`) and not using the includePaths functionality in node-sass?

**Comment by tysongach** [comment]
@msecret Instead of `node-bourbon` and `node-neat`, you should be able to use our official npm packages: `bourbon` and `bourbon-neat`. We actually just released Neat 1.7.3 today, which now has the `includePaths` baked in. We’re also part of [eyeglass](https://github.com/sass-eyeglass/eyeglass), if that makes anything easier.

I’m not sure I fully understand your issue. To clarify, this all works here, but another project is having path issues? Can you point me to that project? Honestly, I’m not too familiar with the npm side of things, but happy to take a peek, and find help if I can’t!

**Comment by msecret** [comment]
Sure I'll try and explain in some more detail.

So I have a separate repo, https://github.com/18F/cg-style, that pulls in the standards with npm, and the @imports the src scss files:

```main.scss
@import "../../node_modules/uswds/src/stylesheets/all.scss";
...
```

When this `main.scss` file is run through sass in the cg-style project, it complains that it can't find bourbon and neat anymore because their being reference by their names and cg-style didn't manipulate includePaths

```uswds/all.scss
@import "bourbon";
@import "neat";
```

I can look at the direct `bourbon` and `bourbon-neat` packages. I also like the idea of eyeglass as this may be something missing in sass dependency management.
